UndertaleModTool/Scripts/Builtin Scripts/DeltaHATE.csx

```diff
@@ -116,7 +117,16 @@ void StringSwap(int n, int k)
     Data.Strings[n].Content = value;
 }
 
-if (Data.GeneralInfo.Filename.Content.ToLower().Contains("undertale"))
+string GameName = Data.GeneralInfo.DisplayName.Content.ToLower();
+bool deltamode = false;
+if (GameName.Contains("undertale") || GameName.Contains("nxtale"))
+    deltamode = false;
+else if (GameName.Contains("survey_program"))
+    deltamode = true;
+else
+    deltamode = ScriptQuestion("Is this Deltarune Chapter 1 or a mod thereof?") ? true : false;
```

Comment:
`ScriptQuestion` already returns a bool, why do a `? true : false` after it?

Reply:
Somehow I was unsure it was a bool, but was sure it could be used in a ternary. In retrospect, of course it's a bool. I'll fix that.